syntax = "proto3";

message Material {
  string key = 1;
  string shortkey = 2;
  string safekey = 3;
  repeated string ncmat_header = 4;
  string dump = 5;
  string ncmat_contents = 6;
  string plot_filename_xsect = 7;
  string extra_keywords = 8;
}

message Database {
  repeated Material materials = 1;
}